src/components/DocExplorer.js

```diff
@@ -56,7 +56,10 @@ export class DocExplorer extends React.Component {
   constructor() {
     super();
 
-    this.state = { navStack: [] };
+    this.state = {
+      navStack: [],
+      searchValue: null
```

Comment:
Exactly; I would like the search action to be a part of navStack - something like:
navStack = [
  testType: { ... },
  graphiql_search: { value: ... },
];